### PR TITLE
Add gif support

### PR DESCRIPTION
**Pull request by cemonel**

### CODE REVIEW

File: prism/app.py [hunk 1]
```diff
@@ -52,21 +52,30 @@ def dispatch_request(self, request):
...
+
     def main(self, request):
         path = request.path[1:]
         _, extension = os.path.splitext(path.lower())
-        if extension not in ('.png', '.jpg', '.jpeg'):
+        if extension not in ('.png', '.jpg', '.jpeg', '.gif'):
             raise NotFound()
+        if extension == '.gif' and request.args.get('out', 'gif') == 'gif':
+            customer = self.get_customer(request)
+            return redirect(core.get_s3_url(customer.read_bucket_name, customer.read_bucket_region, path))
```
Comment (fergalwalsh) [inline prism/app.py]: Does this need to be up here? Can it be below, just above ` if args['command'] == 'info':`?

Comment (cemonel) [reply prism/app.py]: Yeah, that was the first place I put it, but here `args = parse_args(path, request)` function does some validations like `width or height is required`. For **basic** `.gif` case we don't need that.

If the duplicate declaration of `get_customer()` method is the concern, we may call it before the `if extension == '.gif' and request.args.get('out', 'gif') == 'gif':` line. But, here I just don't wanted to break the existing flow of calling the `get_customer` method if all validations are done successfully.

Comment (fergalwalsh) [reply prism/app.py]: > validations like width or height is required. For basic .gif case we don't need that.

We don't actually want to change this behavior. At least not for the gif case specifically. We just want gifs to work with the same parameters as passed for any other image. Right now a width or height is required so the same should be true when serving gifs, even if they do nothing.

Comment (cemonel) [reply prism/app.py]: Ok, right, I will move it as you suggested.

**Review by fergalwalsh — APPROVED** [review]
Looks good! Thank you.

**Pull request merged**